### PR TITLE
Rudimentary start of K0  test descriptions in README.md files

### LINKED ISSUE #11361: [GeomechanicsApplication] Putting documentation in Kratos repository of tests for K0

### PR DESCRIPTION
**Pull request by WPK4FEM**
In the geomechanics team, it was discussed to describe test purpose and results in README.md files like is done for tutorials. This is a first attempt for the K0 procedure process tests.

### CODE REVIEW

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+- [test k0 procedure normal consolidation with layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_k0_nc_layers/README.md)
+- [test k0 procedure normal consolidation with OCR](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_k0_nc_ocr/README.md)
+- [test k0 procedure normal consolidation with skew layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_k0_nc_skew_layers/README.md)
+- [test k0 procedure normal consolidation with skew layers and dam](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_k0_nc_skew_layers_dam/README.md)
+- [test k0 procedure with umat](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_k0_nc_umat/README.md)
+- [test k0 procedure for simple dike](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_simple_dike/README.md)
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/README.md]:
```suggestion
- [K<sub>0</sub>procedure normal consolidation](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md)
- [K<sub>0</sub>procedure normal consolidation with layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_layers/README.md)
- [K<sub>0</sub>procedure normal consolidation with OCR](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md)
- [K<sub>0</sub>procedure normal consolidation with skew layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md)
- [K<sub>0</sub>procedure normal consolidation with skew layers and dam](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md)
- [K<sub>0</sub>procedure with umat](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_umat/README.md)
- [K<sub>0</sub> procedure for simple dike](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md)
```

Comment (mcgicjn2) [reply applications/GeoMechanicsApplication/tests/test_k0_procedure_process/README.md]: The links do not point to the correct location / suggestion should change that

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure simple dike
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K0 procedure process. In the first stage the underground vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. For this a nonlinear ( Mohr Coulomb ) constitutive law is used. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0_NC and checked at an integration point near the bottom of the column. In the second stage, a dam is built on top of the soil. Stresses far away from the dam locatation should not drastically change.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. In the first stage the underground vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. For this a nonlinear ( Mohr Coulomb ) constitutive law is used. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0,NC</sub> and checked at an integration point near the bottom of the column. In the second stage, a dam is built on top of the soil. Stresses far away from the dam location should not drastically change.
```

Comment (mcgicjn2) [reply applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md]: typo

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_layers/README.md [hunk 3]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation with layers
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K0 procedure process. Vertical effective stresses in the column of soil of soil layers follow from soil layer density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0_NC and checked at an integration point near the bottoms of the soil layers.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_layers/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil of soil layers follow from soil layer density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0,NC</sub> and checked at an integration point near the bottoms of the soil layers.
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K0 procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0_NC and OCR and checked at an integration point near the bottom of the column.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0,NC</sub> and OCR and checked at an integration point near the bottom of the column.
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md [hunk 5]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A straightforward test for the K0 procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0_NC and checked at an integration point near the bottom of the column.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md]:
```suggestion
A straightforward test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0,NC</sub> and checked at an integration point near the bottom of the column.
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md]:
```suggestion
# Test K<sub>0</sub> procedure normal consolidation
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation with skew layers
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md]:
```suggestion
# Test K<sub>0</sub> procedure normal consolidation with skew layers
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md [hunk 8]
```diff
@@ -0,0 +1,6 @@
...
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K0 procedure process. Vertical effective stresses in the column of soil of soil layers follow from soil layer density, gravity constant, vertical distance to the surface and the presence of pore water pressure. The soil layers are positioned at an angle deviating from horizontal. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0_NC and checked at an integration point near the bottoms of the soil layers. As the layers deviate from horizontal, the resulting horizontal stress field is not in equilibrium.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil of soil layers follow from soil layer density, gravity constant, vertical distance to the surface and the presence of pore water pressure. The soil layers are positioned at an angle deviating from horizontal. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0,NC</sub> and checked at an integration point near the bottoms of the soil layers. As the layers deviate from horizontal, the resulting horizontal stress field is not in equilibrium.
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md [hunk 9]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation with skew layers and a dam on top
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md]:
```suggestion
# Test K<sub>0</sub> procedure normal consolidation with skew layers and a dam on top
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md [hunk 10]
```diff
@@ -0,0 +1,6 @@
...
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K0 procedure process. Vertical effective stresses in the column of soil of soil layers and a dam follow from soil layer density, gravity constant, vertical distance to the surface and the presence of pore water pressure. The soil layers are positioned at an angle deviating from horizontal. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0_NC and checked at an integration point near the bottoms of the soil layers. As the layers deviate from horizontal and there is a dam weighting those down, the resulting horizontal stress field is not in equilibrium.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil of soil layers and a dam follow from soil layer density, gravity constant, vertical distance to the surface and the presence of pore water pressure. The soil layers are positioned at an angle deviating from horizontal. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0,NC</sub> and checked at an integration point near the bottoms of the soil layers. As the layers deviate from horizontal and there is a dam weighting those down, the resulting horizontal stress field is not in equilibrium.
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md [hunk 11]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure with umat
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md]:
```suggestion
# Test K<sub>0</sub> procedure with umat
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md [hunk 12]
```diff
@@ -0,0 +1,6 @@
...
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K0 procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K0 = 1 - sin(Phi) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md [hunk 13]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure simple dike
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md]:
```suggestion
# Test K<sub>0</sub> procedure simple dike
```

**Review by mcgicjn2 — CHANGES_REQUESTED** [review]
Mainly typos and structure.

We need to add an issue about making our test suite documentation better

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md [hunk 14]
```diff
@@ -0,0 +1,6 @@
+# Test k0 procedure normal consolidation
```
Comment (WPK4FEM) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md]:
```suggestion
# Test K<sub>0</sub> procedure normal consolidation
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/README.md [hunk 15]
```diff
@@ -0,0 +1,12 @@
...
+- [K<sub>0</sub>procedure normal consolidation with OCR](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md)
+- [K<sub>0</sub>procedure normal consolidation with skew layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md)
+- [K<sub>0</sub>procedure normal consolidation with skew layers and dam](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md)
+- [K<sub>0</sub>procedure with umat](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_umat/README.md)
+- [K<sub>0</sub> procedure for simple dike](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md)
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/README.md]:
```suggestion
- [K<sub>0</sub> procedure normal consolidation](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc/README.md)
- [K<sub>0</sub> procedure normal consolidation with layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_layers/README.md)
- [K<sub>0</sub> procedure normal consolidation with OCR](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_ocr/README.md)
- [K<sub>0</sub> procedure normal consolidation with skew layers](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers/README.md)
- [K<sub>0</sub> procedure normal consolidation with skew layers and dam](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_skew_layers_dam/README.md)
- [K<sub>0</sub> procedure with umat](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_nc_umat/README.md)
- [K<sub>0</sub> procedure for simple dike](https://github.com/KratosMultiphysics/applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_simple_dike/README.md)
```

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md [hunk 16]
```diff
@@ -0,0 +1,6 @@
+# Test K<sub>0</sub> procedure with umat
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi;) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```

**Review by mcgicjn2 — CHANGES_REQUESTED** [review]

File: applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md [hunk 17]
```diff
@@ -0,0 +1,6 @@
+# Test K<sub>0</sub> procedure with umat
+
+**Author:** [Wijtze Pieter Kikstra](https://github.com/WPK4FEM)
+
+## Case Specification
+A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi;) and checked at an integration point near the bottom of the column. Phi is retrieved from the parameter list of the user supplied material.
```
Comment (mcgicjn2) [inline applications/GeoMechanicsApplication/tests/test_k0_procedure_process/test_k0_procedure_k0_umat/README.md]:
```suggestion
A test for the K<sub>0</sub> procedure process. Vertical effective stresses in the column of soil follow from density, gravity constant, vertical distance to the surface and the presence of pore water pressure. Horizontal effective stresses are derived from the vertical effective stresses by multiplication with K<sub>0</sub> = 1 - sin(&Phi;) and checked at an integration point near the bottom of the column. Phi (&Phi;) is retrieved from the parameter list of the user supplied material.
```

**Review by mcgicjn2 — CHANGES_REQUESTED** [review]

**Review by mcgicjn2 — APPROVED** [review]

**Pull request merged**